docs/_includes/Statements.md

```diff
@@ -375,10 +425,14 @@ WhileStmt = "while"
 
 ````
 WhileAlternativeBlock =
-   "{" { "case" Expression(allowLemma: true, allowLambda: false) "=>" { Stmt } } "}" .
+   "{" 
+   { "case" Expression(allowLemma: true, allowLambda: false) 
+   "=>" { Stmt } } 
+   "}
 ````
 
-See section [#sec-loop-specification] for a description of ``LoopSpec``.
+Loops may also need _loop specifications_ (``LoopSpec`` in the grammar) in order for Dafny to prove that
```

Comment:
I don't like "may also need". I see loop specifications as a feature, since it allows you to reason about loops and to describe what is intended for them to do. How about something like "may contain" or "can include"?

Reply:
Reworded.